new_platforms/vsextension/ProjectWizard/ImportEnclaveCommand.cs

```diff
@@ -441,7 +441,12 @@ private async void Execute(object sender, EventArgs e)
                             if (gdbRule != null)
                             {
                                 // Configure GDB debugger settings.
-                                gdbRule.SetPropertyValue("PreLaunchCommand", "export PYTHONPATH=/opt/openenclave/lib/openenclave/debugger/gdb-sgx-plugin;export LD_PRELOAD=/opt/openenclave/lib/openenclave/debugger/liboe_ptrace.so");
+                                string gdbEnvironmentSettings = "export PYTHONPATH=/opt/openenclave/lib/openenclave/debugger/gdb-sgx-plugin;export LD_PRELOAD=/opt/openenclave/lib/openenclave/debugger/liboe_ptrace.so";
```

Comment:
These are oegdb internals. It would be preferable not to rely on the internals here. Is it possible to invoke the oegdb wrapper script  that hides all the internals.

Reply:
/opt/openenclave is the default install location for OE. However, an user could install OE to any location or even have multiple installs if say they are trying to debug an issue that occurs only in a particular release of OE. Can we avoid this assumption about OE install location?

Reply:
I don't believe VS lets you change the debugger executable name, you can only run scripts before invoking gdb, and commands sent to gdb.  So I could not find any way to invoke the oegdb wrapper.

Reply:
it is possible to avoid assumption about OE install location in the future by prompting the user.  This would be more work though so will not be in this changelist.

Reply:
> I don't believe VS lets you change the debugger executable name, you can only run scripts before invoking gdb, and commands sent to gdb. So I could not find any way to invoke the oegdb wrapper.

Created #1905 for future work.